google/cloud/bigtable/client_options_test.cc

```diff
@@ -300,6 +302,13 @@ TEST(ClientOptionsTest, SetSslTargetNameOverride) {
             grpc::string(test_args.args[1].key));
 }
 
+TEST(ClientOptionsTest, UserAgentPrefix) {
+  std::string const actual = bigtable::ClientOptions::UserAgentPrefix();
+
+  EXPECT_THAT(actual, HasSubstr("cbt-c++/"));
+  EXPECT_THAT(actual, ::testing::AnyOf(HasSubstr(" noex"), HasSubstr(" ex")));
```

Comment:
Should we look for " noex " or " ex " instead (i.e., blanks on both sides)?

Reply:
Done.